linux_os/guide/services/ssh/ssh_server/sshd_set_max_sessions/rule.yml

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Set SSH MaxSessions limit'
+
+description: |-
+    The <tt>MaxSessions</tt> parameter specifies the maximum number of open sessions permitted
+    from a given connection. To set MaxSessions edit
+    <tt>/etc/ssh/sshd_config</tt> as follows: <pre>MaxSessions <b>sessions</b></pre>
+
+rationale: |-
+    To protect a system from denial of service due to a large number of concurrent
+    sessions, use the rate limiting function of MaxSessions to protect availability
+    of sshd logins and prevent overwhelming the daemon.
+
+severity: medium
+
+references:
+  cis@rhel8: 5.2.19
+  cis@sle15: 5.2.23
+
+ocil_clause: "MaxSessions is configured to the default of 10"
```

Comment:
Unfortunately, this is not correct either. The original was fine. So, either revert to the original, or use the following:

```
MaxSessions is not configured or not configured correctly
```

Unfortunately, at this time, `idref` is not available in the `ocil_clause` making it harder to correctly set this.

Reply:
0c6d42e